courses/free-courses-en.md

```diff
@@ -504,6 +505,12 @@
 * [Learn Redux](https://learnredux.com) - Wes Bos (email address *requested*)
 
 
+#### Svelte
+
+* [Learn the Svelte JavaScript Framework](https://www.youtube.com/watch?v=ujbE0mzX-CU) - Noah Glaser, freeCodeCamp (YouTube)
+* [Svelte Tutorial](https://www.youtube.com/watch?v=vhGiGqZ78Rs) - Beau Carnes, freeCodeCamp (YouTube)
```

Comment:
56min with chapters integrated in timeline

🤔 Too short?

Reply:
Short, but quite dense. The course covers essentially building a whole app and a good amount of Svelte APIs. Happy to remove this if it doesn't fit!

Reply:
It may also be a good fit for "Screencasts". Let me know which you'd prefer and I'll make the adjustment.